src/rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Kindly make sure that `book` and `person` has getters and setters by adding them to `attr_accessor` instead
```suggestion
  attr_accessor :date, :book, :person
```